src/pages/ProfilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,199 @@
+import * as yup from 'yup';
+
+import EditButtons from './components/EditButtons/EditButtons';
+import FormRow from './components/FormRow/FormRow';
+import TextInput from './components/TextInput/TextInput';
+import styles from './ProfilePage.module.css';
+
+const REQUIRED_ERROR = 'This field is required.';
+const SPECIAL_CHARACTERS_ERROR = 'No special characters allowed.';
+
+const schema = yup.object({
+  name: yup
+    .string()
+    .required(REQUIRED_ERROR)
+    .matches(/^[A-Za-z0-9 ]+$/, SPECIAL_CHARACTERS_ERROR),
+  password: yup.string(),
+});
+
+const ProfilePage = () => {
+  const [selectedTab, setSelectedTab] = useState(0);
+  const theme = useTheme();
+  const [formData, setFormData] = useState({ nameEditable: false, passwordEditable: false, tempImage: '' });
+
+  const {
+    register,
+    getValues,
+    setValue,
+    formState: { errors },
+  } = useForm({
+    mode: 'onBlur',
+    resolver: yupResolver(schema),
+  });
+
+  const handleChange = (event, newValue) => {
+    setSelectedTab(newValue);
+  };
+
+  const avatarUrl = () => {
+    if (formData?.image && typeof formData?.image === 'string') {
+      return formData.image;
+    }
+
+    if (formData?.tempImage instanceof File) {
+      return URL.createObjectURL(formData.tempImage);
+    }
+    return '';
+  };
+
+  const setEditable = (field) =>
+    setFormData((prevState) => ({ ...prevState, [`${field}Editable`]: !prevState[`${field}Editable`] }));
+
+  const onChangeName = () => {
+    setEditable('name');
+    setFormData((prevState) => ({ ...prevState, namePrev: getValues('name') }));
+    // SEND REQUEST
+  };
+
+  const onChangePassword = () => {
+    setEditable('password');
+    setFormData((prevState) => ({ ...prevState, passwordPrev: getValues('password') }));
+    // SEND REQUEST
+  };
+
+  const onCancelChange = (field) => {
+    const fieldPrev = `${field}Prev`;
+
+    if (formData[fieldPrev]) {
+      setValue(field, formData[fieldPrev]);
+      setEditable(field);
+    } else {
+      setEditable(field);
+    }
+  };
+
+  const onAddAvatar = (image) => {
+    setFormData((prevState) => ({ ...prevState, tempImage: image }));
+    // SEND REQUEST
+  };
+
+  const getInitials = () => (getValues('name')?.[0] ? getValues('name')[0].toUpperCase() : undefined);
+
+  return (
+    <div>
+      <Container component={Paper} maxWidth={false} className={styles.container}>
+        <div className={styles.headerContainer}>
+          <Box sx={{ width: '100%', bgcolor: 'background.paper' }}>
+            <Tabs
+              value={selectedTab}
+              onChange={handleChange}
+              textColor="secondary"
+              indicatorColor="secondary"
+              variant="scrollable"
+              scrollButtons="auto"
+              sx={{ maxWidth: 'max-content', margin: '0 auto' }}
+            >
+              <Tab label="Account Settings" />
+              <Tab label="Favourites" />
+              <Tab label="Sell House" href="/sell-house" />
```

Comment:
Will you be adding the tabs to the sell-house page as part of this PR?

Reply:
I did it as an example to get a better feel for the layout. I can do it on my own, but I prefer to create new PR for it - it takes too many changes for one PR.

Reply:
Sure. Just create a new task for this Jira and add it to a relevant sprint.